spec/lib/hubspot/meeting_spec.rb

```diff
@@ -12,6 +12,91 @@
   let(:hs_meeting_start_time) { DateTime.strptime('2022-05-03T10:00:00+01:00', '%Y-%m-%dT%H:%M:%S%z') }
   let(:hs_meeting_end_time) { DateTime.strptime('2022-05-03T10:15:00+01:00', '%Y-%m-%dT%H:%M:%S%z') }
 
+  describe '.all' do
+    let(:options) { {} }
+    let(:expected_ids) do
+      %w[12642321179 12642400564 12643234101 12643330095 12643437384 27195487241 27475250495 27475252583 27475257356]
```

Comment:
this will break if anyone wants to rebuild the cassette. Do we really need it?

Reply:
:shrug: If they need to rebuild the cassette, they might as well rewrite the examples.

Reply:
How do you suggest I test the `after` param then ? Without reusing the cassette values, I don't really see how it's possible.